packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -96,6 +97,7 @@ describe('SdkClient', async function () {
       duration,
     );
 
+    // @ts-ignore
```

Comment:
if this is needed because error is `EventEmitter` is not generic, we might need to update also server's and ws-server's `@types/node` dependency.